### DISCUSSION TITLE
Using Fluid with Angular

### DISCUSSION THREAD

**Discussion by tylerbutler** [microsoft/FluidFramework#5464] [Documentation topics]
~This article isn't written yet, but you can discuss and vote on it here. We prioritize writing articles that receive the most votes.~

This article is now published here: <https://fluidframework.com/docs/recipes/angular/>

**Comment by tedrob** [comment]
I would prefer a angular example,

**Reply by RiggedToBlow** [reply]
@tedrob 
there is a couple of decent examples in this repo https://github.com/DanWahlin/FluidAngular
check it out

**Reply by RichardC64** [reply]
And there too: https://github.com/RichardC64/AngularFluidExamples

**Comment by tylerbutler** [comment]
This article is now published here: <https://fluidframework.com/docs/recipes/angular/>
